torch/csrc/jit/runtime/register_c10_ops.cpp

```diff
@@ -169,16 +169,11 @@ class RegistrationListener final : public c10::OpRegistrationListener {
       // TODO Find a better way to handle this.
       return;
     }
-    if (at::is_aten_op_and_unboxing_is_already_handled_by_c10(
-            op.schema().operator_name())) {
+    if (at::is_aten_op(op.schema().operator_name())) {
```

Comment:
Aside: in a world where the listener setup is no longer needed (because the JIT knows how to do c10 registry lookups directly), where will this conditional creation of a testing wrapper wind up?

Reply:
Excellent question, moving this to do a namespace check instead might solve the issue.